fastestimator/op/tensorop/gradient/gradient.py

```diff
@@ -74,9 +78,23 @@ def forward(self, data: List[Tensor], state: Dict[str, Any]) -> List[Tensor]:
                 results.append(get_gradient(final, initial, tape=state['tape'], retain_graph=retain_graph))
         else:
             finals = data
-            trainable_params = [p for p in self.model.parameters() if p.requires_grad] if isinstance(
-                self.model, torch.nn.Module) else self.model.trainable_variables
-            for idx, final in enumerate(finals):
-                retain_graph = self.retain_graph or not idx == len(finals) - 1
-                results.append(get_gradient(final, trainable_params, tape=state['tape'], retain_graph=retain_graph))
+            if self.framework == "tf":
+                trainable_params = self.model.trainable_variables
+                for idx, final in enumerate(finals):
+                    # get_gradient
```

Comment:
labeling the get_gradient function with a get_gradient comment is not super useful

Reply:
fixed